openshift/performance/patches/001-upstream-tests.patch

```diff
@@ -0,0 +1,1755 @@
+diff --git a/test/performance/README.md b/test/performance/README.md
```

Comment:
Not sure if its really a good idea to patch all of the files in. Is this not going to break all the time and we have to fix that huge git patch? 

I still feel like this is a huge maintenance burden "just" to have elasticsearch. Maybe we should just copy the tests from upstream with the openshift specifics? @nak3 maybe you have a better idea?

Reply:
> Not sure if its really a good idea to patch all of the files in. Is this not going to break all the time and we have to fix that huge git patch?

Half of the patch consists of new files which dont affect anything. Readme files can also be avoided to be patched.
What is left is a few changes in the job yaml files (which should not create conflict unless something drastically changes) and job main changes (this could create some conflicts). In general I doubt there will be much activity around perf tests (as in the past). 

In any case one way to decouple the patch/tests is to maintain a different repo with its own modified tests so conflicts can be resolved independently or have a folder under which we apply the patch only when tests are run and not by default.

Reply:
Yeah I see your points. We can also just see how it progresses. As you say, maybe this is not an issue anyway.
But can we do something different with the go.mod (https://github.com/openshift-knative/serving/pull/505#discussion_r1376152895)? There we are definitely going to conflict pretty fast, right?